language-plutus-core/src/Language/PlutusCore/Lexer/Type.hs

```diff
@@ -53,8 +53,40 @@ data BuiltinName = AddInteger
                  | EqByteString
                  | TxHash
                  | BlockNum
+                 -- See Note [sizeOfInteger].
+                 | SizeOfInteger
                  deriving (Show, Eq, Ord, Enum, Bounded, Generic, NFData, Lift)
 
+{- Note [sizeOfInteger]
+The 'sizeOfInteger' built-in is a later addition. The main motivation for adding it is that it
+allows to pass less singleton sizes around. However less boilerplate is not the only advantage,
```

Comment:
Not sure if you're interested in this kind of remark but: sizes are discrete, boilerplate is continuous, so "fewer singleton sizes" and "less boilerplate".

Reply:
I always appreciate correction like that, thank you.